Makefile

```
@@ -77,7 +77,7 @@ container-push: container-image ## Push container image for the preview of the w
 container-build: module-check
 	$(CONTAINER_RUN) --read-only --mount type=tmpfs,destination=/tmp,tmpfs-mode=01777 $(CONTAINER_IMAGE) sh -c "npm ci && hugo --minify --environment development"
 
-container-serve: module-check ## Boot the development server using container. Run `make container-image` before this.
+container-serve: module-check ## Boot the development server using container.
```

I feel like this explains it better. Please feel free to ignore if the suggestion is not helpful or too verbose.
```suggestion
container-serve: module-check ## Boot the development server using a preview container image pulled from the registry.
```